docs/src/pages/components/geo/index.page.mdx

```diff
@@ -0,0 +1,71 @@
+---
+title: Geo
+---
```

Comment:
Needs a `description` added, and a SEO description and title in the `meta` file:
https://github.com/aws-amplify/amplify-ui/blob/7b13bdad16bb2f78143e972bbdd641b0409daa29/docs/src/data/meta.ts

Reply:
Done